libsql-server/src/namespace/store.rs

```diff
@@ -366,6 +366,18 @@ impl NamespaceStore {
         restore_option: RestoreOption,
         db_config: DatabaseConfig,
     ) -> crate::Result<()> {
+        if let Some(shared_schema_name) = &db_config.shared_schema_name {
+            self.fork(
+                shared_schema_name.clone(),
+                namespace.clone(),
+                db_config.clone(),
+                None,
+            )
+            .await?;
+            let to_store = self.config_store(namespace).await?;
+            return to_store.store(db_config).await;
```

Comment:
I don't think we need this any more once fork takes the db config already, no?

Reply:
you are right. I kept it here thinking we don't flush inside Fork, but we fixed that yesterday. Let me update.